jdat_notebooks/transit_spectroscopy_notebook/Exoplanet_Transmission_Spectra_JWST.ipynb

```diff
@@ -1647,6 +1647,7 @@
     "plt.errorbar(wsd,rprs**2*1E6,xerr=werr,yerr=2*rerr*rprs*1E6, fmt='o',zorder=10,color='orange',label='Recovered Spectra')\n",
     "plt.xlabel('Wavelength ($\\mu$m)')\n",
     "plt.ylabel('Transit Depth ($R_p/R_s$)$^2$ (ppm)')\n",
+<<<<<<< HEAD
```

Comment:
This is what's making the notebook unusable. You need to delete either the code between `<<<<<<< HEAD` and `=======` or `=======` and ` >>>>>>> 5e04021b6b55220c191460a6de0d259412d2a6a6`, depending on what you want to keep, and then delete those three lines of git markup. Committing and pushing after fixing that should fix the problem I was seeing.

Reply:
Weird, it's clearly some copy paste issue, but I didn't show up at at on my local version, until I tried re-opening it, which failed.